reference/config_files/conan.conf.rst

```diff
@@ -134,6 +135,10 @@ as it may result in corrupted packages.
 The ``default_package_id_mode`` changes the way package IDs are computed. By default, if not specified
 it will be ``semver_direct_mode``, but can change to any value defined in :ref:`package_id_mode`.
 
+The ``full_transitive_package_id`` changes the way package IDs are computed regarding transitive dependencies.
```

Comment:
This changes the way if, and only if, there are header-only packages in the graph. I'd add that note.

Reply:
As said above, not only for header-only